dbt-bigquery/tests/functional/adapter/incremental/test_incremental_unique_id.py

```diff
@@ -1,5 +1,146 @@
-from dbt.tests.adapter.incremental.test_incremental_unique_id import BaseIncrementalUniqueKey
+from dbt.tests.adapter.incremental.test_incremental_unique_id import SubBaseIncrementalUniqueKey
```

Comment:
Way to minimally disturb the tests that other adapters depend on while also accounting for differences in semantics from flipping the null equality condition on (and tests for which this fails).